seabeepy/geo.py

```diff
@@ -112,7 +112,7 @@ def upload_raster_to_geoserver(fpath, user, password, workspace="geonode"):
         workspace=workspace,
     )
 
-    return layer_name
+    return None
```

Comment:
we don't have to return anything to return None

Reply:
Yeah, I know. I like to explicitly return `None` in cases where the return value is likely to be used/tested, but in this case I just lazily changed `layer_name` to `None`. I'll remove it

Reply:
:+1:  you will hate [Julia](https://groups.google.com/g/julia-users/c/4RVR8qQDrUg) :grinning:

Reply:
Yep, I already hate Julia! I'm hoping Mojo is what I've been waiting for...